Packs/NationalVulnerabilityDatabaseFeed/Integrations/NationalVulnerabilityDatabaseFeed/NationalVulnerabilityDatabaseFeed.yml

```diff
@@ -0,0 +1,233 @@
+  combines CVE and CPE in a suite of tools to help automate vulnerability management
+  and evaluate compliance with federal information technology security requirements.
+
+  The NVD maintains the authoritative CPE dictionary, while the CVE program is maintained
+  by the MITRE corporation. Both programs are sponsored by the U.S. Department of
+  Homeland Security (DHS) and the Cybersecurity and Infrastructure Security Agency
+  (CISA). The National Vulnerability Database (NVD) is tasked with analyzing each
+  CVE once it has been published to the CVE List. NVD analysts use the reference information
+  provided with the CVE and any publicly available information at the time of analysis
+  to associate Reference Tags, Common Vulnerability Scoring System (CVSS) v2.0, CVSS
+  v3.1, CWE, and CPE Applicability statements.
+
+  CVEs are typically available in the NVD within an hour of their publishing. Once
+  a CVE is in the NVD, analysts can begin the analysis process. The processing time
+  can vary depending on the CVE, the information available, and the quantity of CVEs
+  published within a given timeframe. After analysis is provided, CVEs may be updated
+  (modified). If modifications are available, the NVD publishes these updates once
+  every two hours. The CPE Dictionary is updated nightly when modifications or new
+  names are added.
+
+  ### Request an API Key
+  Navigate to https://nvd.nist.gov/developers/request-an-api-key
+  On the API key requests page, enter data into the three fields on the requests form.
+  Scroll to the bottom of the Terms of Use, and then click the check box marked “I
+  agree to the Terms of Use.”
+  Check the inbox of the email address provided in the steps above for an email from
+  nvd-noreply@nist.gov.
+  Activate and view the API Key by opening the single-use hyperlink. Store the API
+  Key in a secure location as the page will no longer be available after it is closed.
+  If your key is not activated within seven days, a request for a new API Key must
+  be submitted.
+  Each API Key is associated with a single email address. If an email address is used
+  to request an additional API key, clicking the single-use hyperlink will invalidate
+  the key previously associated with that email address.  The key will not be invalidated
+  if the email is used to request another key, but the link is not opened. There is
+  no process for retrieving a forgotten key.
+
+  ### Rate Limits
+  Requesting an API key allows for users to make a greater number of requests in a
+  given time than they could otherwise. The public rate limit (without an API key)
+  is 10 requests in a rolling 60 second window; the rate limit with an API key is
+  100 requests in a rolling 60 second window.
+
+  ---
+configuration:
+- name: apiKey
+  display: API Key
+  required: false
+  type: 4
+- name: type
+  display: Feed Type
+  required: true
+  defaultvalue: CVE
+  type: 15
+  additionalinfo:
+  options:
+  - CVE
+# - CPE
```

Comment:
Is there a reason they need to be removed? For the moment, they are there as a reminder (and simple code change) to allow the collection of CPEs. Once we either have a dedicated object for CPEs, or an indicator type for CPE, it can be simply uncommented.

Reply:
OK, we can keep the comments